plugin/pkg/admission/security/podsecuritypolicy/admission.go

```diff
@@ -57,8 +57,8 @@ func Register(plugins *admission.Plugins) {
 // PSPMatchFn allows plugging in how PSPs are matched against user information.
 type PSPMatchFn func(lister extensionslisters.PodSecurityPolicyLister, user user.Info, sa user.Info, authz authorizer.Authorizer, namespace string) ([]*extensions.PodSecurityPolicy, error)
 
-// podSecurityPolicyPlugin holds state for and implements the admission plugin.
-type podSecurityPolicyPlugin struct {
+// PodSecurityPolicyPlugin holds state for and implements the admission plugin.
+type PodSecurityPolicyPlugin struct {
```

Comment:
Why change this?

Reply:
Because we cannot return one admission interface in the constructor anymore. This was largely discussed within https://github.com/kubernetes/kubernetes/pull/54485. Golang doesn't give us better tools than this.